scripts/setup-gce-vm.sh

```diff
@@ -22,7 +22,8 @@ sudo apt-get update && sudo apt-get install --no-install-recommends -y mariadb-s
 
 # Give all privileges to all databases to the root user on localhost and then
 # reload the privilege grants
-sudo mysql -e "grant all privileges on *.* to 'root'@'localhost' identified by 'password'; flush privileges"
+sudo mysql \
+  -e "grant all privileges on *.* to 'root'@'localhost' identified by 'password'; flush privileges"
```

Comment:
Can we add a note to not use password 'password' in real life.  Please and thanks

Reply:
Sure thing.